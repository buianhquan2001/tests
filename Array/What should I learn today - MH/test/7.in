20
4
22 13 14 6 
5
15 7 24 2 14 
1
13 
26
20 9 12 16 11 6 15 4 4 15 25 5 13 5 3 19 15 10 11 9 15 7 13 20 24 13 
16
24 3 10 5 24 18 14 9 6 7 16 2 18 10 13 24 
25
2 4 24 8 10 21 6 10 15 6 9 21 1 15 9 9 21 23 21 2 11 25 10 10 14 
22
7 19 9 9 7 25 23 14 5 11 20 14 13 3 24 24 6 14 11 13 3 6 
21
19 19 12 21 8 6 10 9 15 2 9 4 25 10 1 17 23 25 17 4 6 
14
19 4 23 8 15 9 13 13 15 12 25 5 17 12 
25
11 7 14 9 3 14 11 9 10 16 15 7 14 4 6 23 16 4 12 11 14 22 24 9 8 
19
6 23 2 3 3 23 23 2 18 13 14 15 8 6 2 8 24 21 10 
21
6 15 13 18 14 17 15 20 25 25 10 24 23 23 12 5 14 21 8 5 2 
9
2 22 19 1 11 14 24 11 21 
4
8 24 25 15 
12
4 24 16 12 22 20 7 23 19 7 5 2 
9
17 18 25 19 18 24 15 22 4 
24
15 15 3 2 20 2 8 15 19 23 16 22 25 25 10 24 2 16 11 20 24 4 9 4 
15
11 25 23 9 20 19 18 9 19 13 18 9 23 11 2 
19
22 19 25 11 15 8 14 17 9 5 21 23 16 13 7 24 16 17 15 
22
4 17 20 13 20 2 18 17 2 20 24 7 17 14 17 2 14 22 14 25 1 22 
