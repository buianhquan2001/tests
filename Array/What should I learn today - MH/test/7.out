math
physics
math
math
math
physics
math
math
math
math
physics
physics
physics
physics
english
english
english
english
math
english
